securitas/themes/fas/static/js/fedora-bootstrap/fedora-bootstrap.css

```diff
@@ -51,22 +51,18 @@ article, aside, figcaption, figure, footer, header, hgroup, main, nav, section {
 
 body {
   margin: 0;
-  font-family: "Open Sans";
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Why this change?

Reply:
same thing, this basically an bundled library. so i'm -1 for changing this file